src/services/offer-service.ts

```diff
@@ -31,9 +33,16 @@ export const OfferService = {
   getOffer: async (id: string): Promise<AxiosResponse<Offer>> =>
     await axiosClient.get(createUrlPath(URLs.offers.get, id)),
 
-  getUsersOffers: async (params: GetMyOffersParams): Promise<AxiosResponse> => {
+  getUsersOffers: (params: GetMyOffersParams) => {
     const user = createUrlPath(URLs.users.get, params.id)
-    return await axiosClient.get(`${user}${URLs.offers.get}`, { params })
+    const resultUrl = getFullUrl({
+      pathname: `${user}${URLs.offers.get}`,
+      searchParameters: { ...params }
```

Comment:
`getFullUrl` is compatible with nested objects like `sort`
If you will have a typescript error like `Index signature for type 'string' is missing in type`, just change the definition `GetMyOffersParams` from `interface` to a `type`.
```suggestion
      searchParameters: params
```

Reply:
Changed to type, but error still present.

Reply:
Fixed.